languages/python/exercises/concept/none/.docs/introduction.md

```diff
@@ -0,0 +1,32 @@
+In Python, `None` is frequently used to represent the absence of a value -- a placeholder to define a `null` (empty) variable, object, or argument.
+
+If you've heard about or used a `NULL` or `nil` type in another programming language, then this usage of `None` in Python will be familiar to you. `None` helps you to declare variables that you don't yet have values for. These variables can then be re-assigned to a value later as needed.
+
+```python
+a = None
+print(a)
+#=> None
+type(a)
+#=> <class 'NoneType'>
+```
+
```

Comment:
So I think it might be goo to add an example here of `none` as a default value for an argument.  
Something like:

```python
def add_to_todos(new_task, todo_list=None):
  if todo_list is None:
    todo_list = []
    todo_list.append(new_task)
  return todo_list
```